Software/app/basic_lorawan/.project

```diff
@@ -1173,24 +1343,144 @@
 			<locationURI>PARENT-2-PROJECT_LOC/lib/STM32WLxx_LoRaWAN/SubGHz_Phy/stm32_radio_driver/radio_driver.h</locationURI>
 		</link>
 		<link>
-			<name>lib/STM32WLxx_LoRaWAN/SubGHz_Phy/stm32_radio_driver/radio_mw_version.h</name>
+			<name>lib/STM32WLxx_LoRaWAN/SubGHz_Phy/stm32_radio_driver/subghz_phy_version.h</name>
 			<type>1</type>
-			<locationURI>PARENT-2-PROJECT_LOC/lib/STM32WLxx_LoRaWAN/SubGHz_Phy/stm32_radio_driver/radio_mw_version.h</locationURI>
+			<locationURI>PARENT-2-PROJECT_LOC/lib/STM32WLxx_LoRaWAN/SubGHz_Phy/stm32_radio_driver/subghz_phy_version.h</locationURI>
 		</link>
 		<link>
 			<name>target/stm32wlxx_hal_driver/Inc/Legacy/stm32_hal_legacy.h</name>
 			<type>1</type>
 			<locationURI>PARENT-2-PROJECT_LOC/target/stm32wlxx_hal_driver/Inc/Legacy/stm32_hal_legacy.h</locationURI>
 		</link>
 		<link>
-			<name>lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/packages/LmhpCompliance.c</name>
+			<name>lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/Packages/LmhpCompliance.c</name>
+			<type>1</type>
+			<locationURI>PARENT-2-PROJECT_LOC/lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/Packages/LmhpCompliance.c</locationURI>
+		</link>
+		<link>
+			<name>lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/Packages/LmhpCompliance.h</name>
+			<type>1</type>
+			<locationURI>PARENT-2-PROJECT_LOC/lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/Packages/LmhpCompliance.h</locationURI>

```

Comment:
All instances of '.../LmHandler/Packages/...' should be '.../LmHandler/packages/...' since the folder name is lower case. Alternatively you could change the folder name.

Reply:
Fixed